doc/source/assets.rst

```diff
@@ -272,10 +273,11 @@ From this data it extracts the phase of the moon at a given time.
 
 .. ipython:: python
 
-   import requests
    import json
+   import os
+   import requests
    url = "https://api.forecast.io/forecast/{api_key}/{lat},{long},{time}"
-   api_key = "1378698e5711b504f22b32eb1a40d23a"
+   api_key = os.environ['FORECAST_IO_API_KEY']
```

Comment:
Looks like fresh-out-of-grad-school Dan was careless with API keys. I have retired this key, registered for a new key, and configured it on Travis-CI.